StripeConnect/StripeConnect/Source/Internal/Webview/MessageHandlers/Helpers/VoidMessage.swift

```diff
@@ -0,0 +1,10 @@
+//
+//  VoidMessage.swift
+//  StripeConnect
+//
+//  Created by Chris Mays on 8/2/24.
+//
+
+import Foundation
+
+struct VoidMessage: Codable, Equatable { }
```

Comment:
- nit: It might be nice to add a docstring for this
- optional: "Payload" instead of "Message"?
```suggestion
/// Represents an empty message payload (e.g. `{}`)
struct VoidMessagePayload: Codable, Equatable { }
```